src/drive/locales/en.json

```
@@ -444,6 +444,8 @@
     "alert": {
       "success": "%{smart_count} file uploaded with success. |||| %{smart_count} files uploaded with success.",
       "success_conflicts": "%{smart_count} file uploaded with %{conflictNumber} conflict(s). |||| %{smart_count} files uploaded with %{conflictNumber} conflict(s).",
+      "success_updated": "%{smart_count} file uploaded and %{updatedCount} updated(s). |||| %{smart_count} files uploaded and %{updatedCount} file(s) updated.",
```

```suggestion
      "success_updated": "%{smart_count} file uploaded and %{updatedCount} updated(s). |||| %{smart_count} files uploaded and %{updatedCount} updated.",
```

"5 files uploaded and 2 updated"… just a suggestion to avoid the file(s).